0002-capture-matching-next-steps.md

```diff
@@ -0,0 +1,115 @@
+
+## Considered Options
+
+1. Implement mobile application side image collection improvements, including enforced phone rotation, enforced compass bearing, and perhaps others.
+2. Research feasibility of leveraging autocorrelation between GPS measurements to improve selection of candidates for location based matching of captures.
+3. Create experimental manual capture matching UI that offers manual spatial analysis relevant to disambiguating matching captures.
+4. Research approaches to image analysis that can produce a similarity metric to improve selection of candidates for location based matching of captures.
+5. Attempt to train models using machine learning based on existing database of capture images to identify repeat captures.
+6. Incorporate assigned species information during capture matching.
+7. Create curated training sets of matched trees for to support training of machine learning based models.
+<!--
+## Decision Outcome
+
+Chosen option: "{option 1}", because {justification. e.g., only option, which meets k.o. criterion decision driver | which resolves force {force} | … | comes out best (see below)}.
+
+### Positive Consequences <!-- optional -->
+
+* {e.g., improvement of quality attribute satisfaction, follow-up decisions required, …}
+* …
+
+### Negative Consequences <!-- optional -->
+
+* {e.g., compromising quality attribute, follow-up decisions required, …}
+* …
+-->
+
+## Pros and Cons of the Options
+
+### 1. Implement mobile application side image collection improvements, including enforced phone rotation, enforced compass bearing, and perhaps others.
+
+Currently, images captured on the mobile application do not have specific criteria guiding capture.  Thus, they exist at a variety of camera rotation angles and compass bearings, even for the same tree being repeat captured.  This requires a manual operator to consider images of the same tree that almost always exist at different rotations and focal distances, greatly enhancing the difficulty of an already challenging manual task.
+
+Constraints can be implemented on the mobile side to ameliorate this situation.  For instance, camera rotation around the x axis could be constrained to 0, 45, and 90 degrees.  Additionally, compass bearing could be constrained such that captured photos are always north facing.  Implementing a constraint on focal distance is also possible, as has been identified in other apps.  
+
+* Good, because the images captured will have more constitant parameters.
+* Good, because this only affect the mobile application and does not require data research or algorithm development.
+* Bad, because the mobile application will become more complex for our low technical literacy users, and we will have to invest careful UX/UI development time to make these features easily usable.
+
+### 2. Research feasibility of leveraging error autocorrelation between GPS measurements to improve selection of candidates for location based matching of captures.
```

Comment:
This option seems to be the right path to go down, if we can get the dedicated resource required.
If we have sufficient markers in the data such as this, we may be able to calculate a "match confidence" score and present it to the user.

Reply:
I have done some preliminary analysis work on this, I have some math that takes into account the 'travel vector' or a grower as they move between trees.

Reply:
Initial analysis and algorithm work for the travel vector approach can be found here: https://github.com/Greenstand/treetracker-location-analysis/blob/main/travel-vector/spatial.ipynb